src/tools/r2rdump/R2RMethod.cs

```diff
@@ -43,7 +43,9 @@ class RuntimeFunction
         /// </summary>
         public R2RMethod Method { get; }
 
-        public RuntimeFunction(int id, int startRva, int endRva, int unwindRva, R2RMethod method)
+        public UnwindInfo UnwindInfo { get; }
+
+        public RuntimeFunction(R2RReader r2r, int id, int startRva, int endRva, int unwindRva, R2RMethod method)
```

Comment:
Could you pass in the UnwindInfo object as a parameter instead of having to pass in the R2RReader just to create the UnwindInfo?